include/specfem_mpi/specfem_mpi.hpp

```diff
@@ -224,8 +227,9 @@ class MPI {
   void bcast(double &val, int root) const;
 
 private:
-  int world_size; ///< total number of MPI processes
-  int my_rank;    ///< rank of my process
+  int world_size;  ///< total number of MPI processes
+  int my_rank;     ///< rank of my process
+  int extern_init; ///< flag to check if MPI was initialized outside SPECFEM
```

Comment:
This should be a `bool`

Reply:
MPI_Initialized() takes int pointer as argument.